package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "marvin",
+  "version": "0.0.0",
+  "description": "An API driven skin for MediaWiki",
+  "scripts": {
+    "start": "run-p -n dev:*",
+    "dev:client": "NODE_ENV=development webpack -w",
+    "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
+    "prod:build": "NODE_ENV=production webpack -p",
+    "format": "prettier --write 'src/**/*.js'",
+    "lint": "eslint 'src/**/*.js'",
+    "test": "npm run lint",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "npm run test"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/niedzielski/marvin.git"
+  },
+  "keywords": [
+    "mediawiki",
+    "wikimedia",
+    "web",
+    "node",
+    "pwa"
+  ],
+  "author": "Wikimedia Foundation",
+  "license": "GPL-2.0",
+  "bugs": {
+    "url": "https://phabricator.wikimedia.org/project/profile/2878/"
+  },
+  "homepage": "https://phabricator.wikimedia.org/project/profile/2878/",
+  "dependencies": {
```

Comment:
Let's talk about this later but we  are going to be using npm@5, which now has package-lock.json which actually fixes all the installed deps to the exact version. Like shrinkwrap did in previous npm versions. As you can see in the files changed, I've committed the package-lock.json to the repo, as recommended by npm inc., so that everyone npm installing will get **exactly** the same versions of the dependencies.

We will only upgrade by explicitly running `npm update` and then commiting the modified package-lock.json.

Very cool new feature from npm@5!! 🎉